src/lexer.rs

```diff
@@ -160,6 +101,7 @@ fn granularize(block: &str) -> Result<Vec<Token>> {
             ")" => CloseRound,
             "?" => Question,
             "-" => Dash,
+            "=" => Assignment,
```

Comment:
Done. Replaces the existing `==` clause with `={1,2}` which should still preferentially match `==`, but will still match `=`.